backend/app/cmd/server.go

```diff
@@ -494,28 +500,29 @@ func (s *ServerCommand) newServerApp() (*serverApp, error) {
 	}
 
 	srv := &api.Rest{
-		Version:            s.Revision,
-		DataService:        dataService,
-		WebRoot:            s.WebRoot,
-		RemarkURL:          s.RemarkURL,
-		ImageProxy:         imgProxy,
-		CommentFormatter:   commentFormatter,
-		Migrator:           migr,
-		ReadOnlyAge:        s.ReadOnlyAge,
-		SharedSecret:       s.SharedSecret,
-		Authenticator:      authenticator,
-		Cache:              loadingCache,
-		NotifyService:      notifyService,
-		SSLConfig:          sslConfig,
-		UpdateLimiter:      s.UpdateLimit,
-		ImageService:       imageService,
-		EmailNotifications: emailNotifications,
-		EmojiEnabled:       s.EnableEmoji,
-		AnonVote:           s.AnonymousVote && s.RestrictVoteIP,
-		SimpleView:         s.SimpleView,
-		ProxyCORS:          s.ProxyCORS,
-		AllowedAncestors:   s.AllowedHosts,
-		SendJWTHeader:      s.Auth.SendJWTHeader,
+		Version:             s.Revision,
+		DataService:         dataService,
+		WebRoot:             s.WebRoot,
+		RemarkURL:           s.RemarkURL,
+		ImageProxy:          imgProxy,
+		CommentFormatter:    commentFormatter,
+		Migrator:            migr,
+		ReadOnlyAge:         s.ReadOnlyAge,
+		SharedSecret:        s.SharedSecret,
+		Authenticator:       authenticator,
+		Cache:               loadingCache,
+		NotifyService:       notifyService,
+		SSLConfig:           sslConfig,
+		UpdateLimiter:       s.UpdateLimit,
+		ImageService:        imageService,
+		EmailNotifications:  emailNotifications,
+		TelegramBotUsername: telegramNotificationsBotUsername,
```

Comment:
this looks like java ;) tgNotifBotName maybe?

Reply:
Got rid of this helper variable in the second commit for good.